core/commands/files/files.go

```diff
@@ -61,7 +63,24 @@ var FilesStatCmd = &cmds.Command{
 	Arguments: []cmds.Argument{
 		cmds.StringArg("path", true, false, "Path to node to stat."),
 	},
+	Options: []cmds.Option{
+		cmds.StringOption("format", "Print statistics in given format. Allowed tokens: "+
+			"<hash> <size> <cumulsize> <type> <childs>. Conflicts with other format options.").Default(
+			`<hash>
```

Comment:
It was always like that, changing it would break CLI compatibility.

It is so you could do `ipfs files stat /file | head -1` to get the hash before format options were a thing.